src/OrbitQt/CallTreeWidget.h

```diff
@@ -56,6 +57,7 @@ class CallTreeWidget : public QWidget {
   void onCopyKeySequencePressed();
   void onCustomContextMenuRequested(const QPoint& point);
   void onSearchLineEditTextEdited(const QString& text);
+  void onSearchTypingFinishedTimerTimout();
```

Comment:
This isn't very related to the PR, and obviously it's consistent with the rest of the class, but I'm wondering if we've decided on going with QTs coding style for slots instead of our usual style (starting with upper-case)? I think we're doing this differently in different widgets.

Reply:
Eh, good question. I think that in some cases Qt induced us to use its style, and then this propagated for consistency.

Reply:
I renamed them, clang-tidy was complaining anyway.